specification/metrics/semantic_conventions/container.md

```diff
@@ -0,0 +1,129 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: HTTP
+--->
+
+# Semantic Conventions for Container Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## HTTP Server
+
+### Metric: `metric.container.cpu.usage`
```

Comment:
The name should just be `container.cpu.usage`

The `metric.` classifier is for our YAML database, not the metric name. (Sorry for confusion)

Same comment for all other metrics.